### PR TITLE
Fix of documentation

### PR DESCRIPTION
**Pull request by gordonguthrie**

### CODE REVIEW

**Comment by gordonguthrie** [comment]
Cheeky wee PR for @javajolt

File: README.md [hunk 1]
```diff
@@ -43,10 +43,10 @@ The shell is in the early stages. The following are well supported:
 * batch mode
 * management of connections to remote riak nodes
 * shell management (including cookies)
+* integration with riak_test so that replay logs can be run as simple regression tests
```
Comment (hazen) [inline README.md:46]: OK. I guess I can live with that. Let's get 'er in.

**Comment by hazen** [comment]
:+1: 0306132

**Comment by hazen** [comment]
YOLO merge

**Pull request merged**